webproj/api.py

```diff
@@ -3,44 +3,34 @@
 import json
 from pathlib import Path
 
-from flask import Flask
-from flask_cors import CORS
-from flask_restx import Api, Resource, abort
+import uvicorn
+from fastapi import FastAPI
+from fastapi.responses import JSONResponse
+from fastapi.middleware.cors import CORSMiddleware
+
 import pyproj
 from pyproj.transformer import Transformer, AreaOfInterest
 
-from webproj.utils import IntFloatConverter
-
 version = "1.2.0"
 
 if "WEBPROJ_LIB" in os.environ:
     pyproj.datadir.append_data_dir(os.environ["WEBPROJ_LIB"])
 
 # Set up the app
-app = Flask(__name__)
-app.url_map.converters["number"] = IntFloatConverter
-CORS(app)
-
-api = Api(
-    app,
+app = FastAPI(
+    title=__name__,
+    description="## API til koordinattransformationer\n\nAPIet "
+                "__WEBPROJ__ giver adgang til at transformere "
+                "multidimensionelle koordinatsæt. \n\nTil adgang "
+                "benyttes Dataforsyningens brugeradgang som ved andre "
+                "tjenester.\n\n[Versionshistorik](/webproj.txt)",
     version=version,
-    title="WEBPROJ",
-    description=(
-        "## API til koordinattransformationer"
-        "\n\n"
-        "API'et __WEBPROJ__ giver adgang til at transformere "
-        "multidimensionelle koordinatsæt."
-        "\n\n"
-        "Til adgang benyttes Dataforsyningens brugeradgang som ved andre "
-        "tjenester."
-        "\n\n"
-        "[Versionshistorik](/webproj.txt)"
-    ),
```

Comment:
Hvad er fordelen ved at omformatere dette?

Reply:
Ikke noget synderligt. Tror det er sket ved copy/paste fra gammelt til nyt format, og så har jeg ikke tjekket ordentligt efter. Ændrer det tilbage, så det er lidt mere overskueligt

Reply:
Og ja næste skridt må være tests. Men er nok også nødt til at omskrive test-scriptet, for det kører på nogle forældede metoder.